src/services/__tests__/pagoPAProxyService.test.ts

```diff
@@ -177,6 +182,24 @@ describe("PagoPAProxyService#getPaymentInfo", () => {
     expect(res.kind).toEqual("IResponseErrorValidation");
   });
 
+  it("getPaymentInfoDefaultDecoder must validate correctly a PaymentProblemJson response", async () => {
+    const a = {
+      detail: PaymentFaultEnum.DOMAIN_UNKNOWN
+    };
+    const validator = getPaymentInfoDefaultDecoder();
+    const res = mockRes();
+    // tslint:disable: no-object-mutation
```

Comment:
don't disable globally, use `disable-next-line` instead

Reply:
fixed in 35258de